requirements.txt

```diff
@@ -9,3 +9,9 @@ frozendict>=2.3.4
 beautifulsoup4>=4.11.1
 html5lib>=1.1
 peewee>=3.16.2
+
+# Sphinx Documentation
```

Comment:
Not essential for a user install so shouldn't be in this file. At minimum, comment-out.